src/components/pages/Rockect.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/prop-types */
```

Comment:
- Proptypes is a mechanism that ensures that the passed value is of the correct datatype. I think you should not disable the Linter, you can solve it manually.

Reply:
done